e2e_test/v2/batch/generate_series.slt

```diff
@@ -11,6 +11,23 @@ select generate_series from generate_series('2008-03-01 00:00:00'::TIMESTAMP,'20
 2008-03-04 12:00:00
 
 query II
+select * from generate_series(timestamp '2020-01-01 01:02:03', timestamp '2021-01-01 03:04:05', interval '1' month);
+----
+2020-01-01 01:02:03
+2020-02-01 01:02:03
+2020-03-01 01:02:03
+2020-04-01 01:02:03
+2020-05-01 01:02:03
+2020-06-01 01:02:03
+2020-07-01 01:02:03
+2020-08-01 01:02:03
+2020-09-01 01:02:03
+2020-10-01 01:02:03
+2020-11-01 01:02:03
+2020-12-01 01:02:03
+2021-01-01 01:02:03
+
+query III
```

Comment:
I guess that there should be only one `I`, but I'm not sure why there is no error reported.

```suggestion
query I
```

Reply:
The `I` represents one Integer column here.